lib/simulation/openfisca/mapping/reverse.js

```diff
@@ -4,10 +4,17 @@ var periods = require('./periods');
 var PRESTATIONS = require('../prestations');
 
 
-module.exports = function reverseMap(openFiscaFamille, date) {
+module.exports = function reverseMap(openFiscaFamille, date, injectedRessources) {
```

Comment:
This is getting really hard to grasp. Add a [JSdoc comment](http://usejsdoc.org/tags-param.html) to describe the parameters of this function.